charts/pega/README.md

```diff
@@ -12,7 +12,7 @@ k8s         | Open-source Kubernetes
 openshift   | Red Hat Openshift
 eks         | Amazon Elastic Kubernetes Service (EKS)
 gke         | Google Kubernetes Engine (GKE)
-pks         | Pivotal Container Service (PKS)
+pks         | VMware Tanzu Kubernetes Grid Integrated Edition (TKGI)
```

Comment:
Should we update the value so that `pks || tkgi` results in the same deployment configuration?

Reply:
I don't know. I updated both sections. Let me know if this makes sense now.